upstairs/src/lib.rs

```diff
@@ -1595,6 +1595,8 @@ async fn test_return_iops() {
  */
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct Arg {
+    /// Upstairs session UUID
+    pub session_id: String,
```

Comment:
It's not really important, but you can keep this a UUID if you'd prefer. It'll be `serde`-serialized in that case.